Gems/Atom/RPI/Code/Source/RPI.Private/RPISystemComponent.cpp

```diff
@@ -93,20 +96,30 @@ namespace AZ
             }
 
             m_rpiSystem.Initialize(m_rpiDescriptor);
-            AZ::SystemTickBus::Handler::BusConnect();
+            AZ::TickBus::Handler::BusConnect();
```

Comment:
comment: As mentioned in my comment, I am worried this could break Track View render out capture (@VickyAtAZ might be able to comment on this too). Definitely something to check (and I'll add to my list adding a test for this as it would be nice if we could catch it breaking automatically).